endpoints/events/event.go

```diff
@@ -104,11 +104,19 @@ func (e *eventEndpoint) Handle(w http.ResponseWriter, r *http.Request, _ httprou
 		return
 	}
 
-	// account does not support events
-	if !account.EventsEnabled {
-		w.WriteHeader(http.StatusUnauthorized)
-		w.Write([]byte(fmt.Sprintf("Account '%s' doesn't support events", eventRequest.AccountID)))
-		return
+	// account does not support events and events are not enabled at instance level (account config takes precedence)
+	if account.EventsEnabled != nil {
+		if !*account.EventsEnabled {
+			w.WriteHeader(http.StatusUnauthorized)
+			w.Write([]byte(fmt.Sprintf("Account '%s' doesn't support events", eventRequest.AccountID)))
+			return
+		}
+	} else {
+		if !e.Cfg.Event.Enabled {
```

Comment:
Nitpick: can we put the conditional in an `else if` to avoid nesting?
```
117 -   } else {
118 -       if !e.Cfg.Event.Enabled {
    +   } else if !e.Cfg.Event.Enabled {
119         w.WriteHeader(http.StatusUnauthorized)
120         w.Write([]byte(fmt.Sprintf("Account '%s' or server doesn't support events", eventRequest.AccountID)))
121         return
122 -       }
123     }
endpoints/events/event.go
```